apps/forms/form-dialog-alert/src/app/ui/dialog.component.ts

```diff
@@ -1,24 +1,34 @@
 import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { MatDialogClose } from '@angular/material/dialog';
 
 // NOTE : this is just the dialog content, you need to implement dialog logic
 
 @Component({
   standalone: true,
+  imports: [MatDialogClose],
   template: `
-    <div role="alert" class="rounded-xl border border-gray-100 bg-white p-5">
-      <h3 class="block text-xl font-medium text-red-600">
-        You have unsaved information!
+    <div
+      role="alertdialog"
+      aria-labelledby="alert-title"
+      aria-describedby="alert-description"
```

Comment:
The `aria-labelledby` attribute is generally the id of the element used to title the alertdialog.
The `aria-describedby` attribute is generally the ID of the element containing the alert dialog's messaging, usually coming right after the title.

Reply:
All interactive elements must have an accessible name. aria-labelledby can be used to reference another element to define its accessible name, when an element's accessible name needs to use content from elsewhere in the DOM. - [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby).

This leads to believe that you should just use `aria-label` on each button of the modal instead.